x/gov/proposals.go

```diff
@@ -7,12 +7,13 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/proposal"
 )
 
 // Proposal is a struct used by gov module internally
 // embedds ProposalContent with additional fields to record the status of the proposal process
 type Proposal struct {
-	ProposalContent `json:"proposal_content"` // Proposal content interface
+	proposal.Content `json:"proposal_content"` // Proposal content interface
```

Comment:
in my experience. writing `proposal.proposal_content` it terrible UX for clients
```suggestion
	proposal.Content `json:"content"` // Proposal content interface
```

Reply:
++